tests/test_search.py

```diff
@@ -38,29 +39,30 @@ class ESInfo:
 
 
 def setUpModule():
+    IndexSuffix.name = __name__.split('.')[-1]
     ESInfo.server = ElasticsearchServer()
     os.environ['DSS_ES_PORT'] = str(ESInfo.server.port)
 
 
 def tearDownModule():
     ESInfo.server.shutdown()
-
+    IndexSuffix.reset()
+    os.unsetenv('DSS_ES_PORT')
 
 class TestSearchBase(DSSAsserts):
     @classmethod
     def search_setup(cls, replica):
         cls.replica_name = replica.name
-        cls.dss_index_name = dss.Config.get_es_index_name(dss.ESIndexType.docs, replica)
         dss.Config.set_config(dss.DeploymentStage.TEST)
+        cls.dss_index_name = dss.Config.get_es_index_name(dss.ESIndexType.docs, replica)
         cls.app = dss.create_app().app.test_client()
         with open(os.path.join(os.path.dirname(__file__), "sample_index_doc.json"), "r") as fh:
             cls.index_document = json.load(fh)
 
     def setUp(self):
         dss.Config.set_config(dss.DeploymentStage.TEST)
         self.app = dss.create_app().app.test_client()
-        elasticsearch_delete_index(self.dss_index_name)
-        create_elasticsearch_index(self.dss_index_name, logger)
+        elasticsearch_delete_index(f"*{IndexSuffix.name}")
```

Comment:
why was the `create_elasticsearch_index(..)` call moved to the specific test?

Reply:
The index is created when the first bundle is indexed. Since no bundles are indexed the index is never created. In all other test cases bundles are indexed before searching.

Reply:
What's the harm of leaving the create_index call in `setUp`?

Reply:
it adds unnecessary steps, otherwise it's harmless.